src/content/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file.mdx

```diff
@@ -1209,6 +1209,43 @@ Set these options in the `common` section. To [override](#System_Properties) any
       enabled: true
     ```
   </Collapser>
+
+  <Collapser
+    id="trim_stats"
+    title="trim_stats"
+  >
+    <table>
+      <tbody>
+      <tr>
+        <th>
+          Type
+        </th>
+
+        <td>
+          Boolean
+        </td>
+      </tr>
+
+      <tr>
+        <th>
+          Default
+        </th>
+
+        <td>
+          `true`
+        </td>
+      </tr>
+      </tbody>
+    </table>
+
+    When the exclusive time of a segment is less than 2% of the total transaction time, the metric for that method call will be aggregated under `Java/other`.
+
+    Setting `trim_stats` to `false` will disable this behavior.
```

Comment:
Can we also add/referencing that disabling this behavior will cause the actual metric to be displayed or something like that?

Would like to make it clear to customers what the benefit/value of disabling it is.